common/src/main/java/com/wynntils/core/consumers/overlays/TextOverlay.java

```diff
@@ -123,27 +119,4 @@ private float getTextScale() {
     protected abstract String getTemplate();
 
     protected abstract String getPreviewTemplate();
-
-    public final boolean isRendered() {
-        // If the enabled template is empty,
-        // the overlay is rendered when the player is in the world.
-        String template = enabledTemplate.get();
-        if (template.isEmpty()) return isRenderedDefault();
-
-        // If the enabled template is not empty,
-        // the overlay is rendered when the template is true.
-        String formattedTemplate =
-                StyledText.join("", Managers.Function.doFormatLines(template)).getString();
-        ErrorOr<Boolean> enabledOrError = Managers.Function.tryGetRawValueOfType(formattedTemplate, Boolean.class);
-        return !enabledOrError.hasError() && enabledOrError.getValue();
-    }
-
-    /**
-     * Returns whether the overlay is rendered with the default (empty) template.
-     *
-     * @return whether the overlay is rendered with the default (empty) template
-     */
-    public boolean isRenderedDefault() {
-        return Models.WorldState.onWorld() && !Models.WorldState.inCharacterWardrobe();
```

Comment:
For a bunch or overlays you removed the `!inCharacterWardrobe` check. Why?

Reply:
I made a better check, it basically checks if you are riding a display entity which happens during these moments which includes character wardrobe